### DISCUSSION TITLE
Desktop version: Cannot set start/finish/intermediate points

### DISCUSSION THREAD

**Discussion by RudderDuck** [organicmaps/organicmaps#716] [Q&A]
I have built the desktop version for Ubuntu 20.04 according to:

https://github.com/LaoshuBaby/organicmaps/blob/master/docs/INSTALL.md

I can start the program, the interface seems fine but I cannot set the start/finish/intermediate points. I can choose between start/finish/intermediate point using the drop-down box but cannot set a point on the map to be start/finish/intermediate point. So I cannot plan a route.

**Comment by vng** [comment]
Desktop UI is not obvious, right:
- choose Start in drop-down box
- use Shift + left mouse button to set point
- choose End in drop-down box
- use Shift + left mouse button to set point
